src/autoscan.cc

```diff
@@ -95,8 +95,7 @@ int AutoscanList::_add(const std::shared_ptr<AutoscanDirectory>& dir)
 {
```

Comment:
Please make it `void`. its return value is not used anywhere.

Reply:
It's used in add above.